draft-ietf-quic-transport.md

```diff
@@ -3830,20 +3813,20 @@ To start ECN validation, an endpoint SHOULD do the following when sending
 packets on a new path to a peer:
 
 * Set the ECT(0) codepoint in the IP header of early outgoing packets sent on a
-  new path to the peer {{!RFC8311}}.
+  new path to the peer ({{!RFC8311}}).
 
 * If all packets that were sent with the ECT(0) codepoint are eventually deemed
-  lost {{QUIC-RECOVERY}}, validation is deemed to have failed.
+  lost ({{QUIC-RECOVERY}}), validation is deemed to have failed.
 
 To reduce the chances of misinterpreting congestive loss as packets dropped by a
 faulty network element, an endpoint could set the ECT(0) codepoint for only the
 first ten outgoing packets on a path, or for a period of three RTTs, whichever
 occurs first.
 
 Other methods of probing paths for ECN support are possible, as are different
-marking strategies. Implementations MAY use other methods defined in RFCs; see
-{{?RFC8311}}. Implementations that use the ECT(1) codepoint need to perform ECN
-validation using ECT(1) counts.
+marking strategies. Implementations MAY use other methods; see {{?RFC8311}}.
```

Comment:
The defined in RFCs is a restatement of the requirements in RFC 8311 that I was specifically requested to include.  I know that it is clumsy, but it's an important point (and apparently a touchy one).